packages/wonder-blocks-switch/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@khanacademy/wonder-blocks-switch",
+  "version": "0.0.1",
+  "design": "v1",
+  "description": "Switch components for Wonder Blocks.",
+  "main": "dist/index.js",
+  "module": "dist/es/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
```

Comment:
**issue:** This needs to be changed to `restricted`, otherwise the release to npm will fail.

https://github.com/Khan/wonder-blocks/actions/runs/5765545001/job/15631699707#step:5:24